### PR TITLE
Update Npm to Yarn

### PR DESCRIPTION
**Pull request by link1697**
Update Npm to Yarn

### CODE REVIEW

File: bootup/kubernetes/templates/postgres/deploy.tmpl [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
     spec:
       containers:
       - name: postgres
-        image: postgres:11.1
+        image: postgres:12
```
Comment (yeasy) [inline bootup/kubernetes/templates/postgres/deploy.tmpl:14]: I remember this k8s plugin is not used anymore?
Can you confirm?

**Review by xichen1 — COMMENTED** [review]
Could you fix the dependency conflict raised by [CICD](https://github.com/hyperledger/cello/actions/runs/11356628303/job/31588156763?pr=647#step:5:15013)?

**Comment by yeasy** [comment]
https://github.com/hyperledger/cello/wiki/Contribution-Process

**Comment by yeasy** [comment]
Upgrade to latest dependency

**Pull request closed without merging**